src/StatusQ/Controls/StatusProgressBar.qml

```diff
@@ -34,10 +34,24 @@ ProgressBar {
             radius: 5
 
             StatusBaseText {
+                id: textItem
                 anchors.centerIn: parent
+                property bool _fittedInBar: width < bar.width ? 1 : 0
```

Comment:
Suggestion: I think we always use true or false for bool types, so maybe better to change them like this for consistency.

Reply:
fixed